src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1047,21 +1112,29 @@ QDateTime TorrentHandle::completedTime() const
         return QDateTime();
 }
 
-int TorrentHandle::timeSinceUpload() const
+std::chrono::seconds TorrentHandle::timeSinceUpload() const
```

Comment:
As I've pointed out before, I think all these functions should return a `duration`, not a specific unit.